src/user-agent/index.ts

```diff
@@ -0,0 +1,402 @@
+export const userAgentData = {
```

Comment:
As property names the compiler cannot minify these variables, which could take a lot of space. Instead we can let them be scoped within the module:

```js
export let geckoDate = 0;
export let mozVersion = -1;
...etc...
```

The way the compiler could turn these into minified variable names.